concepticondata/conceptlists.tsv

```diff
@@ -1,4 +1,5 @@
 ID	AUTHOR	YEAR	LIST_SUFFIX	ITEMS	TAGS	SOURCE_LANGUAGE	TARGET_LANGUAGE	URL	REFS	PDF	NOTE	PAGES	ALIAS
+Serva-2012-200	Serva, Maurizio  and Petroni, Filippo and Volchenkov, Dima  and Wichmann, Søren	2012		200	basic	English	Malagasy dialects	http://people.disim.univaq.it/~serva/languages/languages.html	Serva2012		This list which resembles Swadesh's list of [200 items](:ref:Swadesh-1952-200) closely, was compiled for a study of several varieties of Malagasy dialects. Given that we only had the concept labels for linking, and nobody of the current team knows much about Malagasy dialects, the linking may be sub-optimal at times, and if users detect errors or have suggestions on refinements, we would be very glad if they contacted us.		
```

Comment:
Shouldn't the URL be the one for the meaning list at http://people.disim.univaq.it/~serva/languages/english.html ?